backend/app/views/questions/show.slim

```diff
@@ -0,0 +1,32 @@
+h1
+  = @question.name
+h5
+  | This is a 
```

Comment:
Ops, just realized that we should add translations here as well.
we can also let it go as this and create an issue for it